### ISSUE TITLE
Channel number cannot be selected

### ISSUE THREAD

**Issue by dimmas375** [zhum/ardunoo#1]
_noo.channel()_ function is useless, because all functions work with channel number 255 (-1). Look at your **ardunoo.h** under _public_ declarations.

**Comment by zhum** [comment]
That's not true. All `-1` values are default (and used if you do not specify this argument when do function call) and this value means _use value, which were set already_. Default channel should be set in constructor and can be changed with `channel` function. If you don't provide argument, `channel` function just returns current channel.

**Comment by dimmas375** [comment]
Good to hear.
But your example compiled under Arduino 1.6.7 is not working as you said. Channel number is not changing and 'print current channel' returns 255 regardless of numbers entered before.
Maybe it is compiler's bug, but I have no time to dig into this - I have only one module and it is already installed in my device. Just noted that behavior and used your library as communication sample. Thank you for that.

**Comment by zhum** [comment]
Did you compile my example as-is? No changes? If yes, you should get `5` in response of `c` command in serial console.
Which model are you using?

**Comment by dimmas375** [comment]
It was Uno, your example was compiled to bind the module to power switch.
I've compiled it as is, and during binding process noted that device was binded not on 1st channel as I entered. After that I've tried to change channels and found that the response on 'c' was the same - 255. I took your command buffer as example, and wrote own on/off functions for hardware serial, because the module works with Mega now.
I can't check your code right now - to do that I need go to my summer house, remove the module from working house controller and test it. I do not plan to do that in the near future.
